src/migrations/20190826132443-create-Role.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
why did you delete this migraiton file?

Reply:
I only renamed it.

Reply:
Why will you rename a migration file? It can result in issues when running migrations for other developers.

Reply:
I never knew this